app/models/game.rb

```diff
@@ -5,3 +5,46 @@ class Game < ActiveRecord::Base
   delegate :kings, :queens, :bishops, :knights, :rooks, :pawns, to: :pieces
 
 end
+class Game < ActiveRecord::Base
```

Comment:
Should not redefine the game class.  This should be tucked into the same game class a the top.